### PR TITLE
[BUG #2881] Migrated wallet main screen to latest mockups

### LINKED ISSUE #2881: Update wallet main screen
### User Story

As a user, I want to have access to latest wallet  UI changes 

### Description

Update wallet main screen so that main actions are displayed as a list. `transaction history` is now a first class action.
[Mockups](https://app.zeplin.io/project/5a55d587ba304eed4d2c942f/screen/5a589a0fccc89d66c86d6be8)

*Type*: Feature

### PR DESCRIPTION
**Pull request by jeluard**
fixes #2881

### Summary:

Adapt wallet main screen to latest mockups

### Steps to test:
- Open Status
- Navigate wallet tab
- Verify it matches mockups

EDIT: mocks to check with: https://app.zeplin.io/project/5a55d587ba304eed4d2c942f/screen/5a589a0fccc89d66c86d6be8
status: ready

### CODE REVIEW

File: Jenkinsfile.parameters [hunk 1]
```diff
@@ -41,7 +41,6 @@ node ('macos1') {
       sh 'echo STUB_STATUS_GO='            + STUB_STATUS_GO            + '>>' + '.env'
       sh 'echo ETHEREUM_DEV_CLUSTER='      + ETHEREUM_DEV_CLUSTER      + '>>' + '.env'
       sh 'echo MAINNET_NETWORKS_ENABLED='  + MAINNET_NETWORKS_ENABLED  + '>>' + '.env'
-      sh 'echo ERC20_ENABLED='             + ERC20_ENABLED             + '>>' + '.env'
```
Comment (flexsurfer) [inline Jenkinsfile.parameters]: do we have bounty issue for this?

File: src/status_im/translations/ja.cljs [hunk 2]
```diff
@@ -124,7 +124,7 @@
    :move-to-internal-failure-message      "いくつかの重要なファイルを外部メディアに保存します。続行するには、許可が必要です。（将来的には外部メディアを使用しないようにします。）"
    :debug-enabled                         "デバッグサーバーを起動しました！ *status-dev-cli scan*を実行すると、同じネットワーク上のコンピュータからサーバーを見つけることができます。"
 
-　 ;;phone types
+   　 ;;phone types
```
Comment (flexsurfer) [inline src/status_im/translations/ja.cljs]: indentation

File: src/status_im/ui/components/colors.cljs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+(ns status-im.ui.components.colors)
```
Comment (flexsurfer) [inline src/status_im/ui/components/colors.cljs:1]: Hooray

File: src/status_im/ui/components/colors.cljs [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+(def white "#4360df")
```
Comment (flexsurfer) [inline src/status_im/ui/components/colors.cljs]: if we started this, can we add descriptions to colours? where they used for example

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/components/status_bar/styles.cljs [hunk 5]
```diff
@@ -82,7 +82,7 @@
 
 (defstyle view-wallet
   {:ios     (create-view-style {:background-color styles/color-blue4})
-   :android (create-view-style {:background-color styles/color-blue5})})
+   :android (create-view-style {:background-color styles/color-blue4})})
```
Comment (flexsurfer) [inline src/status_im/ui/components/status_bar/styles.cljs]: `defstyle` isn't needed here , no difference in styles

File: src/status_im/ui/screens/wallet/main/styles.cljs [hunk 6]
```diff
@@ -26,9 +26,7 @@
   {:background-color styles/color-blue4})
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/main/styles.cljs]: if we have colors, why still use styles ?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/components/colors.cljs (hunk 4 above)
Comment (flexsurfer) [reply src/status_im/ui/components/colors.cljs]: maybe we could get comments from design team? for example this colour for buttons, or for text, or for foreground etc

File: src/status_im/ui/screens/wallet/main/styles.cljs (hunk 6 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/main/styles.cljs]: i mean in this ns, we have colors

File: src/status_im/translations/ja.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/translations/ja.cljs]: :+1:

File: Jenkinsfile.parameters (hunk 1 above)
Comment (jeluard) [reply Jenkinsfile.parameters]: Yes, should not be there

File: src/status_im/ui/components/status_bar/styles.cljs (hunk 5 above)
Comment (jeluard) [reply src/status_im/ui/components/status_bar/styles.cljs]: :+1:

File: src/status_im/ui/screens/wallet/main/styles.cljs (hunk 6 above)
Comment (jeluard) [reply src/status_im/ui/screens/wallet/main/styles.cljs]: :+1:

Comment (jeluard) [reply src/status_im/ui/screens/wallet/main/styles.cljs]: I plan to do this PR by PR

File: src/status_im/ui/components/colors.cljs (hunk 4 above)
Comment (jeluard) [reply src/status_im/ui/components/colors.cljs]: :+1: WIP

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
----------------
Branch: PR-3048
Android: https://i.diawi.com/RvMEyC
iOS: https://i.diawi.com/aWsNcP

based on https://app.zeplin.io/project/5a55d587ba304eed4d2c942f/screen/5a589a0fccc89d66c86d6be8 mocks please check the following observations:

iOS:
1. Add total value percentage indicator
2. Add delimiters for Send/Receive/transacton history
3. Add delimiters for Assets
4. Add arrows

image


Android:
1. Add total value percentage indicator
2. Add arrows

image

**Comment by jeluard** [comment]
@asemiankevich We decided not to add the percentage value. @denis-sharypin can you update the mockups?

**Comment by asemiankevich** [comment]
----------------
Branch: PR-3048
Android: https://i.diawi.com/ZDqHQz
iOS: https://i.diawi.com/NbJ7aR

iOS looks fine

image

For Android we need to remove the circles and thats it
image

**Comment by asemiankevich** [comment]
Okay, for the last UI chnages we have cirlces for both platforms. and no precentage label for MVP.

**Pull request merged**